src/components/message/message-card/MessageCardBase.tsx

```diff
@@ -68,25 +69,40 @@ export function MessageCardBase({
 }: MessageCardBaseProps) {
 	const { messageId, timestamp, messageType, sessionId, direction, bodyBase64, body } = message;
 
+	const iconsClassName = createBemBlock(
+		'message-card-icons',
+		isAttached ? 'attached' : null,
+		isBookmarked ? 'pinned' : null,
+	);
+
 	const renderInlineMessageInfo = () => {
 		if (viewType === MessageViewType.ASCII || viewType === MessageViewType.JSON) {
 			const formattedTimestamp = formatTime(timestampToNumber(timestamp));
+			const splittedMessageId = messageId.split(':');
+			const messageIdWithHighlightedSession = splittedMessageId.map(
+				splittedMessageId.length > 3
```

Comment:
instead of using this ternary with 2 functions, your function can accept array which .map provides 3rd argmunent